.github/workflows/ci.yml

```diff
@@ -78,6 +78,22 @@ jobs:
         cargo check --target thumbv6m-none-eabi --features print-defmt
         cargo check --target thumbv6m-none-eabi --features print-rtt
 
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          profile: minimal
+          components: clippy
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
(why) is this needed? (also, I don't think we have set any secrets for this repo.)

Reply:
This is what the [`clippy-check`](https://github.com/actions-rs/clippy-check/#example-workflow) action requires and suggest in its default configuration. 

This step even fails, when this toke key is not set. I guess, it's about a hidden token generated by github, which has to be passed to clippy-check, so that it can annotate PRs with clippy warnings? TBH, I haven't looked much further. The correct answer can be probably found at the [`clippy-check`](https://github.com/actions-rs/clippy-check) repository.